app/views/targets/_facebook_logout.html.haml

```diff
@@ -0,0 +1,25 @@
+:javascript
+  window.fbAsyncInit = function() {
+    FB.init({
+      appId      : '1653169284778740',
+      cookie     : true,
+      xfbml      : true,
+      version    : 'v2.12'
+    });
+    FB.AppEvents.logPageView();
+  };
+
+  (function(d, s, id){
```

Comment:
can you refactor the variable names? is really hard to follow the method as it is